Prace_domowe/Praca_domowa1/Grupa4/Hubert_Drążkowski/pd1/Drazkowski_zadanie1.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Można napisać co to jest regresja Poissona, bo potem ludziom, którzy to sprawdzają, a nie wiedzą jest przykro.</p><p> </p><p>Można też robić jakiś lepszy skład tekstu, bo potem ludziom, którzy to sprawdzają jest trudno.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>